internal/agent/makers.go

```diff
@@ -2,23 +2,67 @@ package agent
 
 import (
 	"github.com/quan-to/chevron/internal/config"
+	"github.com/quan-to/chevron/pkg/database/memory"
+	"github.com/quan-to/chevron/pkg/database/rql"
 	"github.com/quan-to/chevron/pkg/interfaces"
+	"github.com/quan-to/chevron/pkg/models"
 	"github.com/quan-to/slog"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports)